web/src/login/KakaoLogin.tsx

```
@@ -0,0 +1,95 @@
+    return (
+        <button
+            className="p-0 bg-transparent border-none hover:scale-105 transition-transform duration-200"
+            onClick={handleKakaoLogin}
+        >
+            <img
+                src={buttonImageSrc}
+                alt="카카오 로그인"
+                className="w-[320px] md:w-[350px] h-auto"
+            />
+        </button>
+    );
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)